onnx2torch/node_converters/slice.py

```diff
@@ -119,7 +126,19 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
 @add_converter(operation_type='Slice', version=11)
 @add_converter(operation_type='Slice', version=13)
 def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint: disable=unused-argument
+    constant_params = []
+    params_names = node.input_values[1:]
+    for name in params_names:
+        try:
+            constant_params.append(get_const_value(name, graph).tolist())
+        except KeyError:
+            constant_params.append([])
```

Comment:
```suggestion
        if name in graph.initializers:
            constant_params.append(get_const_value(name, graph).tolist())
        else:
            constant_params.append([])
```

Reply:
I think `get_const_value` already checks if `name` is in `graph.initializers` or in `graph.value_as_node_output`.

Reply:
The reason I suggested this `if` condition here is to avoid using the `try, except` block.